### DISCUSSION TITLE
flagged issue

### DISCUSSION THREAD

**Discussion by mdAshik922** [community/community#38071] [Other Feature Feedback, Questions, & Ideas] [answered]
How can I resolve **flagged** issues? Please help me.

**Comment by pascalfarrel** [comment]
Try writing an email to support@github.com or accounts@github.com . They'll usually reply and unflag your account within a few days.
Hope this helps

**Reply by mdAshik922** [reply]
thank you

**Comment by jfullstackdev** [comment, marked as answer]
try this one too 

https://support.github.com/contact/account

**Comment by yalla-shoot-smart** [comment]
Hello my friend
Please look at my account also it has been flagged I don't know why
I opened a ticket and no response yet, her number is #1866989

**Comment by yuanzero** [comment]
I also be flagged too. please someone help me
